python/lsst/ts/electrometer/controller.py

```diff
@@ -310,7 +319,10 @@ async def set_range(self, set_range):
     async def prepare_scan(self):
         """Prepare the keithley for scanning."""
         await self.send_command("TST:TYPE RTC;")
-        await self.send_command(self.commands.set_timer(self.mode))
+        # Comment out so that integration_time isn't affected by set_timer
+        # Added in order to experiment with buffer performance issue
+        # await self.send_command(self.commands.set_timer(self.mode))
```

Comment:
Please don't leave commented-out code; it leads to bit rot. If you really want to save it, put it on a branch in github with a tag and you can always get it back, or make it a config option and run both branches in unit tests.